src/scripts/wizarduserpage.js

```diff
@@ -28,10 +28,9 @@ define(["loading", "globalize", "dashboardcss", "emby-input", "emby-button", "em
             data: {
                 Name: form.querySelector("#txtUsername").value
             },
-            url: apiClient.getUrl("Startup/User"),
-            dataType: "json"
```

Comment:
why remove `dataType`?

Reply:
I changed the API in the backend to void. If datatype is set, it tries to parse the response but it's empty